internal/pkg/errors/strings.go

```diff
@@ -45,6 +45,7 @@ const (
 	StartingConsumerMsg      = "Starting Kafka Consumer. Use Ctrl-C to exit."
 	UpdateTopicConfigMsg     = "Updated the following configuration values for topic \"%s\":\n"
 	UpdateTopicConfigRestMsg = "Updated the following configuration values for topic \"%s\" (read-only configs were not updated):\n"
+	OmitTopicCountMsg        = "The topic count will be omitted as Kafka topics for this cluster could not be retrieved: %s"
```

Comment:
IMO, no need to make a variable in a separate package since this string is only used once. Also, it's more idiomatic to use `%v` here (that way we can pass `err` instead of `err.Error()`)

Reply:
Oh nice, yeah I'll switch to %v. As for why I put it here: it's because I'm checking for this string in the integration test, and I don't want to hardcode it twice.